src/libraries/System.Private.Xml/src/System/Xml/Schema/CompiledidEntityConstraint.cs

```diff
@@ -14,8 +15,8 @@ internal class CompiledIdentityConstraint
     {
         internal XmlQualifiedName name = XmlQualifiedName.Empty;
         private readonly ConstraintRole _role;
-        private readonly Asttree _selector;
-        private readonly Asttree[] _fields;
+        private readonly Asttree _selector = null!;
+        private readonly Asttree[] _fields = null!;
```

Comment:
It's used in other files but it only seems to be used in case of error or circular reference - this doesn't seem to be worth changing for the error case.

I'll leave as is